services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
+
+
+User = get_user_model()
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str | None = None,
+    first_name: str = "",
+    last_name: str = ""
+) -> User:
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+        email=email,

```

Comment:
you need to write optional values only if they are present - if you fill out your records with garbage values (like with empty strings here) it may lead to incorrect results in future queries. For example you want to filter items where first/last name is not NULL but it will give you incorrect result since you put empty strings there